gui/build.py

```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
     # create a intaller
+    if IS_WIN:
```

Comment:
I mean you can treat this patch as a windows bundle function.

> python_path = 'C:\\Python27'
> nsis_path = 'C:\\NSIS' 
And, we when bundle be called under Linux system.

> python_path = /usr/lib/python2.7